src/App.tsx

```diff
@@ -1,22 +1,60 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+const USER_ID = 10284;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [sort, setSort] = useState('All');
+  const [hasError, setHasError] = useState(false);
+
+  const loadTodos = async () => {
+    try {
+      const todoList = await getTodos(USER_ID);
+
+      setTodos(todoList);
+    } catch {
+      setHasError(true);
+      setTimeout(() => {
+        setHasError(false);
+      }, 3000);
+    }
+  };
+
+  useEffect(() => {
+    loadTodos();
+  }, []);
+
+  const filteredTodos = todos.filter(todo => {
+    switch (sort) {
+      case 'Active': return !todo.completed;
+      case 'Completed': return todo.completed;
+      default: return todos;
+    }
+  });
+
+  const visibleTodos = filteredTodos;
```

Comment:
So now you have two variables with same value and purpose.
It looks like you can use filteredTodos directly or call it whatever you want at the beginning